scripts/generate-app-config/generateEvidenceConfig.js

```diff
@@ -12,24 +12,14 @@ const getScreenshotPatterns = generatePagesOptions => (
   generatePagesOptions.searchPatterns.reduce((acc, { root }) => {
     const rootPath = root.replace(/[\\]/g, '/');
     acc.push({
-      pattern: `${rootPath}/packages/*/tests/wdio/__snapshots__/reference/**/*.png`,
+      pattern: `${rootPath}/packages/*/{test,tests}/wdio/__snapshots__/reference/**/*.png`,
       // build out a regex for the entrypoint mask.
-      entryPoint: `${rootPath}/packages/[^/]*/tests/wdio/__snapshots__/reference/`,
+      entryPoint: `${rootPath}/packages/[^/]*/(test|tests)/wdio/__snapshots__/reference/`,
     });
     acc.push({
-      pattern: `${rootPath}/tests/wdio/__snapshots__/reference/**/*.png`,
+      pattern: `${rootPath}/{test,tests}/wdio/__snapshots__/reference/**/*.png`,
       // build out a regex for the entrypoint mask.
-      entryPoint: `${rootPath}/tests/wdio/__snapshots__/reference/`,
-    });
-    acc.push({
-      pattern: `${rootPath}/packages/*/test/wdio/__snapshots__/reference/**/*.png`,
-      // build out a regex for the entrypoint mask.
-      entryPoint: `${rootPath}/packages/[^/]*/test/wdio/__snapshots__/reference/`,
-    });
-    acc.push({
-      pattern: `${rootPath}/test/wdio/__snapshots__/reference/**/*.png`,
-      // build out a regex for the entrypoint mask.
-      entryPoint: `${rootPath}/test/wdio/__snapshots__/reference/`,
+      entryPoint: `${rootPath}/(test|tests)/wdio/__snapshots__/reference/`,
```

Comment:
did It matter if it was `( | )` vs `{ | }`?

Reply:
It seems like the ```( | )``` was the only one that worked, unless I need to code the glob pattern differently.